core/tests/behavior/blocking_write.rs

```diff
@@ -118,6 +118,11 @@ pub fn test_blocking_write_with_special_chars(op: BlockingOperator) -> Result<()
         warn!("ignore test for supabase until https://github.com/apache/incubator-opendal/issues/2194 is resolved");
         return Ok(());
     }
+    // Ignore test for atomicdata until TODO-Issue Link addressed.
+    if op.info().scheme() == opendal::Scheme::Atomicdata {
```

Comment:
Skipping all the `*special_chars` tests as AtomicData cannot handle special chars perfectly yet. One can validate by trying to click into the `aa !@#$%^&()_+-=;,.txt` file under https://atomicdata.dev/Folder/xdi4s7bzo9/folder/cva0jdvg42 or try uploading a file with special chars and clicking into it. An error is shown currently.

Maybe we need to create another issue to keep track of this situation and put the link in these code.